src/services/wallet/walletService.test.ts

```
@@ -0,0 +1,46 @@
+  },
+}));
+
+const mockedFindUnique = prisma.user.findUnique as jest.Mock;
+
+describe("walletService", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("returns the user wallet address when it matches the provided address", async () => {
+    mockedFindUnique.mockResolvedValue({ stellarAddress: "GAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAWHF" });
```

ESLint

[error] Replace `·stellarAddress:·"GAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAWHF"` with `⏎······stellarAddress:⏎········"GAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAWHF",⏎···`

(prettier/prettier)